src/components/modal/searchModal/SearchModal.jsx

```diff
@@ -0,0 +1,82 @@
+import { CloseIcon, SearchIcon } from "assets";
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+
+const SearchModal = ({ setShowSearchModal }) => {
+  const navigate = useNavigate();
+  const { allUsers } = useSelector((state) => state.user);
+  const [searchInput, setSearchInput] = useState("");
+  const [filteredUsers, setFilteredUsers] = useState([]);
+
+  useEffect(() => {
+    if (searchInput.trim() !== "")
+      setFilteredUsers(
+        allUsers.filter((user) =>
+          `${user.firstName} ${user.lastName}`
+            .toLowerCase()
+            .includes(searchInput.toLowerCase().trim())
+        )
+      );
+    else setFilteredUsers([]);
+  }, [searchInput]);
+
+  return (
+    <>
+      <div
+        className={`flex items-start justify-center fixed top-0 bottom-0 left-0 right-0 bg-overlay`}
+        onClick={() => setShowSearchModal(false)}
+      >
+        <div
+          onClick={(event) => {
+            event.stopPropagation();
```

Comment:
I wanted to know as there is no event handler for this `onClick`, then why have you added `stopPropogation()`?

Reply:
Right. Had a different modal structure before. Forgot to remove this. Thanks for pointing it out